sandbox/version-control-manager/src/main/java/com/vikas/method_parser/version_control_manager/details/fetcher/RESTCallHandler.java

```diff
@@ -0,0 +1,65 @@
+package com.vikas.method_parser.version_control_manager.details.fetcher;
+
+import javax.ws.rs.core.MultivaluedMap;
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+
+/**
+ * Used to make Rest Service calls.
+ *
+ */
+public class RESTCallHandler {
+
+    /**
+     * To make a HTTP GET request and get String as output.
+     * @param uri : 
+     * 		     URI of the GET service.
+     * @return output of the REST call.
+     */
+    public static String makeGETCall(String uri) {
+    	if(uri == null || uri.isEmpty()) {
+    		System.out.println("URI cannot be null or empty for a REST call.");
+    	}
+    	String output = null;
+		Client client = Client.create();
+		WebResource webResource = client
+		   .resource(uri);
+		ClientResponse response = webResource.accept("application/json")
+                   .get(ClientResponse.class);
```

Comment:
There can be many more scenarios to be handled here(like query params, request body, request header, etc.), we will handle them later down the line.